server/Components/Pages/Search.razor.css

```
@@ -0,0 +1,25 @@
+/* Fixed-height container so the table body scrolls instead of the whole page */
+.search-grid-container {
+    display: flex;
+    flex-direction: column;
+    height: calc(100vh - 12rem);
+    min-height: 400px;
+}
+
+.search-grid-container ::deep .mud-data-grid {
```

Biome (2.4.6)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)